packages/affiliates/libs/bigquery.js

```diff
@@ -0,0 +1,110 @@
+const moment = require("moment");
+const assert = require("assert");
+const lodash = require("lodash");
+
+// Big query class meant for interacting with the ethereum table.
+module.exports = ({ client } = {}) => {
+  assert(client, "requires bigquery client");
+
+  // types of queries we are interested in
+  const queries = {
+    // logs aka events by contract
+    logsByContract(contract, start, end = Date.now(), selection = ["*"]) {
```

Comment:
These are error logs I'm guessing?

Reply:
logs are just another term for block chain events. I guess logs are the more accurate term since its used here in BQ and other libraries.